src/components/NcInputField/NcInputField.vue

```diff
@@ -305,8 +309,8 @@ export default {
 		},
 
 		validateLabel() {
-			if (this.label && !this.labelOutside) {
-				throw new Error('You need to add a label to the textField component. Either use the prop label or use an external one, as per the example in the documentation')
+			if (!this.label && !this.labelOutside) {
+				throw new Error('You need to add a label to the NcInputField component. Either use the prop label or use an external one, as per the example in the documentation.')
 			}
```

Comment:
> As a side note, I would propose to change from throw new Error to Vue.warn, but that's a matter of taste, I guess.

I'd go that way. Otherwise, it could be kind of a breaking change. Now any component's user without labels would have a broken UI with a crashed component. With `warn` that would be just a warn. And it hasn't been documented.

Reply:
Done. Although it is not that critical, since the component still works even after throwing an error. And before, the component was broken when used correctly :see_no_evil: